backbone.localStorage.js

```diff
@@ -56,7 +56,7 @@ function result(object, property) {
 // with a meaningful name, like the name you'd give a table.
 // window.Store is deprectated, use Backbone.LocalStorage instead
 Backbone.LocalStorage = window.Store = function(name, serializer) {
-  if( !this.localStorage ) {
+  if( !this.localStorage() ) {
```

Comment:
this.localStorage is a function, that condition will always be false because the variable this.localStorage is defined to be a function.  I think you are trying to evaluate whether the value that this.localStorage() returns is available.  That function returns window.localStorage.

FYI I came across this because I overrode that function to return sessionStorage.